services/app/src/pages/game/Game.jsx

```diff
@@ -62,7 +66,11 @@ const Game = () => {
   return (
     <>
       {isAnotherSessionActive && <Navigate to={ROUTES.userTaken} />}
-      <GameHeader />
+      <AvatarUploadModal 
+        showAvatarModal={showAvatarModal}
+        closeAvatarModal={() => setShowAvatarModal(false)}
```

Comment:
rename prop to `onClose` you do not need to specify what modal are you closing when passing prop to `AvatarUploadModal` component